### DISCUSSION TITLE
Rec 2020 on GoPro 12

### DISCUSSION THREAD

**Discussion by javapost1** [gopro/labs#575] [General]
I am interested in recording in Rec 2020. I am a Dolby Vision Certified Colorist and I wish to use GoPro footage that is recorded in the biggest color world to use on projects. I've installed the latest GoPro Labs firmware but am unable to access Rec 2020 color space. I have selected the HEVC at the highest bit depth of 10 bits but upon checking the metadata of the recorded files, they are Rec 709 on my computer. Rec 2020 should be the standard for 10 bit HEVC color. Rec 709 is a very limited color world, developed for HDTV around the turn of the century. GoPros should really be capable of Rec 2020 recording in 2023. If this is available, please instruct me how to access Rec 2020 recording. Thanks, 
<img width="411" alt="GX010052_and_New_Discussion_·_gopro_labs" src="https://github.com/gopro/labs/assets/146569551/5707455a-a256-4ffd-8801-abb52090b8cf">
<img width="408" alt="GX010052_and_New_Discussion_·_gopro_labs" src="https://github.com/gopro/labs/assets/146569551/c05ac310-880e-4bb0-9a68-b470b75ab6a2">

**Comment by dnewman-gpsw** [comment]
There is no ready-to-go calibrated Rec2020 color gamut, but instead you can use oMWIDE=1 for the widest gamut the sensor can produce (significantly wider than Rec709.)  The WIDE gamut is sensor native color.  Combined with oMLOGB=400 you can control the transfer function, and oMNR01=1 to disable in-camera noise reduction, allowing you to expand the dynamic range (may need post de-noise.)  Finally oMBITR=150 for increasing the bitrate. These are the popular ways to expand the palette your GoPro can shoot.  Example from a Labs user:  https://www.youtube.com/watch?v=6jpPrOCDZsM

Third parties have created LUTs for the wide gamut, and IDTs, although I haven't used these myself.

**Comment by javapost1** [comment]
Thanks for the feedback. I’ll check this out. Would be great if companies like BMD DaVinci Resolve had an input transform for this. I don’t want to go to display space but to scene gradi g space like DaVinci wide gamut RGB, for instance.

I’ll check out this video and see what I can figure out. Thanks again. Wouldn’t Rec 2020, built in, be a selling feature for GoPro?

Cheers,

Jack Tunnicliffe
Dolby Vision Certified Colourist/Drone Pilot
Java Post Production
P. 306-777-0160<tel:306-777-0160>
C. 306-536-4321<tel:306-536-4321>

On Sep 30, 2023, at 11:31 AM, David Newman ***@***.***> wrote:

﻿

There is no ready-to-go calibrated Rec2020 color gamut, but instead you can use oMWIDE=1 for the widest gamut the sensor can produce (significantly wider than Rec709.) The WIDE gamut is sensor native color. Combined with oMLOGB=400 you can control the transfer function, and oMNR01=1 to disable in-camera noise reduction, allowing you to expand the dynamic range (may need post de-noise.) Finally oMBITR=150 for increasing the bitrate. These are the popular ways to expand the palette your GoPro can shoot. Example from a Labs user: https://www.youtube.com/watch?v=6jpPrOCDZsM

Third parties have created LUTs for the wide gamut, and IDTs, although I haven't used these myself.

**Comment by dnewman-gpsw** [comment]
Some third party IDTs are here https://github.com/xtremestuff/protune-transforms

This developer hasn't offered WIDE support, but does have Protune Flat Native (wide gamut option GoPro has had since HERO2), which is very close.
